Vaadin 24 - NoSuchMethodError MenuBar.addThemeVariants

Vaadin 24.0.0.alpha9 updates MenuBar to implement HasThemeVariant (https://github.com/vaadin/flow-components/pull/4512) which breaks binary compatibility.

Using ErrorWindowAddon with Vaadin 24.0.0.alpha9+ results in NoSuchMethodError
```
Caused by: java.lang.NoSuchMethodError: 'void com.vaadin.flow.component.menubar.MenuBar.addThemeVariants(com.vaadin.flow.component.menubar.MenuBarVariant[])'
	at com.flowingcode.vaadin.addons.gridhelpers.ColumnToggleHelper.createMenuToggle(ColumnToggleHelper.java:96)
```

https://github.com/FlowingCode/GridHelpers/blob/0676d7e8ef1a3d5916aef5d0f8f47067833b31cc/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/ColumnToggleHelper.java#L96

--- a/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/ColumnToggleHelper.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/ColumnToggleHelper.java
@@ -93,7 +93,7 @@ class ColumnToggleHelper<T> implements Serializable {
     Grid<T> grid = helper.getGrid();
 
     MenuBar menuBar = new MenuBar();
-    menuBar.addThemeVariants(MenuBarVariant.LUMO_TERTIARY_INLINE);
+    menuBar.getThemeNames().add(MenuBarVariant.LUMO_TERTIARY_INLINE.getVariantName());
     MenuItem menuItem = menuBar.addItem(VaadinIcon.ELLIPSIS_DOTS_V.create());
     SubMenu subMenu = menuItem.getSubMenu();
 

